seamless-immutable.development.js

```diff
@@ -99,13 +99,12 @@ function immutableInit(config) {
   ]);
 
   function ImmutableError(message) {
-    var err       = new Error(message);
-    // TODO: Consider `Object.setPrototypeOf(err, ImmutableError);`
-    err.__proto__ = ImmutableError;
-
-    return err;
+    this.name = 'MyError';
```

Comment:
I guess this isn't really as intended:

    this.name = 'MyError';

A copy & paste error? Should likely be `ImmutableError`...